doc/ref_model/chapters/chapter08.rst

```diff
@@ -210,14 +209,133 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
 Requirements, Reference Architecture & Industry Standards Intersect
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The Communcations Service Provider is both a provider and consumer of Cloud based services.
+The Communcations Service Provider (CSP) is both a provider and consumer of Cloud based services.
 When the CSP is actings as:
 
 -  consumer, in which case the typical consideration is total cost of ownership as the consumption is to usually to support internal business operations: BSS/OSS systems;
 -  provider of cloud services, through operation of their own cloud or reselling of cloud services, in which case the typical consideration is margin (cost to offer services vs income received).
 
 These two stances will drive differing approaches to how a CSP would look to manage how it interacts within a Multi-Cloud environment.
 
+Single Pane of Glass
+~~~~~~~~~~~~~~~~~~~~
+
+As organisations spread their resources across on-premises, multiple clouds, and the Edge, the need for a single set of tools and processes to manage and operate across these Hybrid, Edge, and Multi-clouds (HEM clouds) is obvious as can be seen form the following simplistic sceanrio. 
+
+Scenario: An opertor has private clouds that it utilises for its workloads.
```

Comment:
@pgoyal01 Is it the initial situation, which will be transformed into A, B or C solutions?